tests/frame_manager_test.go

```diff
@@ -20,7 +20,7 @@ func TestWaitForFrameNavigationWithinDocument(t *testing.T) {
 	}
 	t.Parallel()
 
-	var timeout time.Duration = 5000 // interpreted as ms
+	timeout := 5 * time.Second
```

Comment:
This shouldn't be changed until the wrong timeout conversion is fixed (related to #456).

The reason is that `WaitForNavigation()` expects an integer, which is then converted to ms:
https://github.com/grafana/xk6-browser/blob/508bcb861edcdad7615e53cbf64c3829b4dfd0b3/common/frame_options.go#L676

If you print `o.Timeout` here, you'll see that `5 * time.Second` sets it to `1388h53m20s`.

Last time I took a look at this, it involved changing how the default timeout is set, which affected a bunch of other places, and I never fixed it. Worth taking a look now and seeing if it can be fixed only for `WaitForNavigation()`?

Reply:
I decided against going down another rabbit hole especially as from the playwright documentation this should be taking integers only. 

But if you want I can try to make a fix using https://github.com/grafana/k6/blob/fcbbcd54acb2ee58cf16d6a15c730809a541afe9/lib/types/types.go#L190-L211 which is what k6 does.

Reply:
Nah, let's just leave the timeout as is for now, but it makes sense to use `GetDurationValue()` once we fix this.

I think we should break compatibility with Playwright in this case, since duration strings are much friendlier than unitless integers, but we can discuss that in #456.